src/esssans/isis/io.py

```diff
@@ -77,4 +73,12 @@ def read_xml_detector_masking(
     )
 
 
-providers = (read_xml_detector_masking, to_path)
+def load_run(filename: FilePath[Filename[RunType]]) -> LoadedFileContents[RunType]:
+    return LoadedFileContents[RunType](sc.io.load_hdf5(filename))
+
+
+def load_direct_beam(filename: FilePath[DirectBeamFilename]) -> DirectBeam:
+    return DirectBeam(sc.io.load_hdf5(filename))
+
+
+providers = (read_xml_detector_masking, load_run, load_direct_beam)
```

Comment:
I don't feel these belong here, since they only work with some special files that we made for the docs. `isis.io` should remain clear of that, i.e., contain only functionality that works on "regular" files.